README.md

```diff
@@ -164,7 +155,7 @@ kotlinter {
     indentSize = 4
     reporters = arrayOf("checkstyle", "plain")
     experimentalRules = false
-    disabledRules = emptyArray<String>()
+    disabledRules = emptyArray()
```

Comment:
It seems like passing the type isn't needed anymore. When I tried that on my own, the type was greyed out